File: C#/RBD/RBD/TPC/Msg/DoCommitMessage.cs
// + TODO check

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using RBD.Msg;
using RBD.Communication;

namespace RBD.TPC.Msg
{
    public class DoCommitMessage : TPCMessage
    {
        override
        public void FromBinary(DataInputStream s) //throws IOException 
        {
            base.FromBinary(s);
        }

        override
        public MessageType GetMessageType()
        {
            return MessageType.TPC_DOCOMMIT;
        }

        override
        public void ToBinary(DataOutputStream s) //throws IOException 
        {
            base.ToBinary(s);
        }
    }
}

File: java/SR-RBD/src/ddb/tpc/msg/DoCommitMessage.java
/**
 * 
 */
package ddb.tpc.msg;

import java.io.IOException;

import ddb.communication.DataInputStream;
import ddb.communication.DataOutputStream;
import ddb.msg.MessageType;

/** 
 * <!-- begin-UML-doc -->
 * <!-- end-UML-doc -->
 * @author User
 * @generated "UML to Java (com.ibm.xtools.transform.uml2.java5.internal.UML2JavaTransform)"
 */
public class DoCommitMessage extends TPCMessage {

	@Override
	public void fromBinary(DataInputStream s) throws IOException {
		super.fromBinary(s);
		
	}

	@Override
	public MessageType getType() {
		return MessageType.TPC_DOCOMMIT;
	}

	@Override
	public void toBinary(DataOutputStream s) throws IOException {
		super.toBinary(s);
	}
}